suites/upgrade/infernalis-x/stress-split/9-workload/snaps-many-objects.yaml

```diff
@@ -0,0 +1,16 @@
+meta:
+- desc: |
+   add test details here
```

Comment:
randomized correctness test for rados operations on a replicated pool with snapshot operations